src/3d/qgs3dmapsettings.cpp

```diff
@@ -141,6 +142,8 @@ void Qgs3DMapSettings::readXml( const QDomElement &elem, const QgsReadWriteConte
                 elemExtent.attribute( QStringLiteral( "yMin" ) ).toDouble(),
                 elemExtent.attribute( QStringLiteral( "xMax" ) ).toDouble(),
                 elemExtent.attribute( QStringLiteral( "yMax" ) ).toDouble() );
+
+    mShowExtentIn2DView = elemExtent.attribute( QStringLiteral( "show-in-2d-view" ), QStringLiteral( "0" ) ).toInt();
```

Comment:
```suggestion
    mShowExtentIn2DView = elemExtent.attribute( QStringLiteral( "showIn2dView" ), QStringLiteral( "0" ) ).toInt();
```

There's a lot of exceptions, but predominantly camel case is used in the QGIS xml elements.

Reply:
Indeed there are a lot of exceptions! Done.